src/ccache.cpp

```diff
@@ -237,6 +238,9 @@ guess_compiler(string_view path)
     return CompilerType::nvcc;
   } else if (name == "pump" || name == "distcc-pump") {
     return CompilerType::pump;
+  } else if (name.find("cl") != nonstd::string_view::npos
+             || name.find("clang-cl") != nonstd::string_view::npos) {
```

Comment:
Looking for "clang-cl" is redundant since if "clang-cl" is found in the string then "cl" has already been found in the string.

Perhaps we should check for an exact name instead of doing substring search?

Reply:
Ack.